packages/flame_3d/example/lib/crate.dart

```diff
@@ -14,7 +14,11 @@ class Crate extends MeshComponent {
   @override
   FutureOr<void> onLoad() async {
     final crateTexture = await Flame.images.loadTexture('crate.jpg');
-    mesh.addMaterialToSurface(0, StandardMaterial(albedoTexture: crateTexture));
+    mesh.updateSurfaces((surfaces) {
+      surfaces[0].material = StandardMaterial(
+        albedoTexture: crateTexture,
+      );
+    });
```

Comment:
@wolfenrain I thought of adding this `updateSurfaces` method as well as a simpler alternative for people!